app/models/cassandra/classification.rb

```diff
@@ -0,0 +1,59 @@
+class Cassandra::Classification
+  include Cequel::Record
+
+  key :project_id, :int
+  key :workflow_id, :int
+  key :subject_id, :int
+  key :workflow_version, :int
+  key :user_ip, :inet
+  key :user_id, :int
+  column :created_at, :timestamp
+  column :updated_at, :timestamp
+  column :user_group_id, :int
+  column :completed, :boolean
+  column :gold_standard, :boolean
+  column :expert_classifier, :boolean
+  column :metadata, :text
+  column :annotations, :text
+
+  after_create :save_subject
+
+  def self.from_ar_model(classification)
+    attrs = classification.attributes.dup
+    attrs.delete("id")
```

Comment:
could the panoptes id be used to link a classification with multiple subjects for say a pairwise comparison?